enhancements/ptp-time-integration.md

```diff
@@ -136,17 +135,13 @@ to support necessary network configurations such as creating a bond interface
 for PTP redundancy, assigning IP address for UDP transport mode.
 
 #### PTP4L configuration file
-
 Default ptp4l configuration file (ptp4l.conf) will be used when starting ptp4l
-service. User doesn't need to specify `-f ptp4l.conf` in `ptp4lOpts`, it is
-automatically appended to `ptp4lOpts` by PTP Daemon. The `ptp4l.conf` file
-uses default content delivered in linuxptp package and is installed under
-path `/etc/ptp4l.conf` inside PTP Daemon image. It cannot be changed via
-CRDs exposed in this proposal. However, some of ptp4l configuration options
-in `ptp4l.conf` can be overwritten by specifying certain options in
-`ptp4lOpts`. For example, with `ptp4lOpts` set to `-2 -s`, it overwrites
-`network_transport` and `slaveOnly` options in `ptp4l.conf` which configures
-ptp4l service to use L2 transport and slave only mode.
+service or the user can specify a custom ptp4l configuration using the `ptp4lconf` CR field.
+The ptp4l configuration will be applied to all the `interfaces` in the policy.
```

Comment:
Each node will have only one profile applied, how can we configure multiple interfaces in one profile?